rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/180_locale_dependent_mapping.yml

```diff
@@ -11,26 +11,26 @@
                       date_field:
                           type: date
                           format: "E, d MMM yyyy HH:mm:ss Z"
-                          locale: "de"
+                          locale: "fr"
```

Comment:
why change to french?

Reply:
It doesn't differ between CLDR and COMPAT, so the same tests work on jdk 22 and 23